### PR TITLE
Add custom command column to repos to provide a way to tweak the build process of individual repositories

### PR DESCRIPTION
**Pull request by kunli2**
There is an issue with the injection job failing ([slack message link](https://moderneinc.slack.com/archives/C04J4KJMUTW/p1675685675059979?thread_ts=1675675949.025439&cid=C04J4KJMUTW)).
Repository [OpenRefine](https://openrefine.org/) 's [ingestion job fails](https://jenkins.moderne.ninja/job/ingest/job/OpenRefine_OpenRefine/61/console) on an `antrun` packaging with `genisoimage`,
This issue can be fixed by adding `-Dmaven.antrun.skip=true and -pl !packaging/pom.xml` to the goals to skip some plugins ([a successful run](https://jenkins.moderne.ninja/job/ingest/job/OpenRefine_OpenRefine/75/)).
However, this is not a generic fix, so this PR provides a generic way to support custom commands for individual repositories by adding a column to `repos.csv` for custom commands.
And for maven builds, this custom command will be added to `goals`.

Take `OpenRefine` as an example, this is the row on repos.csv.
```
OpenRefine/OpenRefine,master,11,,maven,,,,-Dmaven.antrun.skip=true -pl !packaging/pom.xml
```

### CODE REVIEW

File: add-repos.sh [hunk 1]
```diff
@@ -64,6 +64,7 @@ do
     BUILD_ACTION=""
     SKIP=""
     SKIP_REASON=""
+    CUSTOM_COMMAND=""
```
Comment (natedanner) [inline add-repos.sh:67]: I think `BUILD_ACTION` already provides this in the seed.groovy

buildAction | Optional | Additional build tool tasks/targets to execute

Comment (kunli2) [reply add-repos.sh:67]: I see your point, so for `OpenRefine`, we just need to add it to the 5th position

from
```
OpenRefine/OpenRefine,master,11,,maven,,,
```
to
```
OpenRefine/OpenRefine,master,11,,maven,-Dmaven.antrun.skip=true -pl !packaging/pom.xml,,
```

Comment (natedanner) [reply add-repos.sh:67]: Yep. I just updated the readme to reflect all the options that are supported.

**Comment by kunli2** [comment]
will use BUILD_ACTION instead in another PR.

File: add-repos.sh (hunk 1 above)
Comment (kunli2) [reply add-repos.sh:67]: Thanks, I closed this one and pulled up https://github.com/moderneinc/jenkins-ingest/pull/42 instead.

**Pull request closed without merging**